auditlog/README.rst

```diff
@@ -1,4 +1,4 @@
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
```

Comment:
Would you use the new README structure? https://github.com/OCA/maintainer-tools/tree/master/template/module
It would be wonderful :smile:

Reply:
@GoodERPJeff it would be great to have this split as mentioned above